graveyard.json

```
@@ -2358,5 +2358,13 @@
     "link": "https://www.androidpolice.com/google-keen-shutdown-area-120-experiment/",
     "description": "Keen was a Pinterest-style platform with ML recommendations.",
     "type": "app"
+  },
+  {
+    "name": "Cloud Source Repositories",
+    "dateOpen": "2018-06-03",
+    "dateClose": "2024-06-17",
+    "link": "https://cloud.google.com/source-repositories/docs/release-notes#May_15_2024",
```

There's a [new announcement](https://cloud.google.com/source-repositories/docs/release-notes#June_17_2024) confirming the closure.

```suggestion
    "link": "https://cloud.google.com/source-repositories/docs/release-notes#June_17_2024",
```